docs/guide/integrations/multistore.md

```diff
@@ -204,3 +244,119 @@ or
   "
 ></router-link>
 ```
+
+
+## Useful _Helpers_
+
+### How to get current store view?
+
+Here is a helper method to get the value of current store view.
+
+You just need to import `currentStoreView` function from `core/lib/multistore` as example follows : 
+
+```js
+
+import { currentStoreView } from '@vue-storefront/core/lib/multistore'
+
+// ... abridged 
+
+
+  return amount: {
+                currency: currentStoreView().i18n.currencyCode,
+                value: method.price_incl_tax
+              }
+
+```
+
+`currentStoreView()` will return the object value you set in `local.json`
+
+
+### How to remove store code from route
+
+When you need to remove `storeCode` from route in such a case you want to route to default store by mapping fallback, use `removeStoreCodeFromRoute` as following example : 
+ 
+```js
+
+import { removeStoreCodeFromRoute } from '@vue-storefront/core/lib/multistore'
+
+// ... abridged 
+
+    url = (removeStoreCodeFromRoute(url.startsWith('/') ? url.slice(1) : url) as string)
```

Comment:
It is worth to mention that it cannot begin with `/`. But the example in code should be much simpler. The current one is too scary

Reply:
Well this one comes from the core file, let me simplify it though